tcc/hsf-tcc-sample/src/main/resources/file.conf

```diff
@@ -0,0 +1,70 @@
+transport {
```

Comment:
The configuration items change a lot, so replace them all，ref https://github.com/seata/seata/tree/develop/script/client/conf